### PR TITLE
Fixes #12986: Add a generic method to test variable existence

### PR DESCRIPTION
**Pull request by Fdall**
https://www.rudder-project.org/redmine/issues/12986

Replacing previous PR: https://github.com/Normation/ncf/pull/794

### CODE REVIEW

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
Commit modified

File: tree/30_generic_methods/condition_from_variable_existence.cf [hunk 1]
```diff
@@ -18,7 +18,9 @@
 
 # @name Condition from variable existence
 # @description Create a condition from the existence of a variable
-# @documentation This bundle will define a class `{condition_prefix}_{variable_name}_{true/false}
+# @documentation This bundle will define a class `{condition_prefix}_{variable_name}_{true/false}`
+# * `{condition_prefix}_{variable_name}_true` if the variable is defined
+# * `{condition_prefix}_{variable_name}_false` if the variable is not defined
 #
 # @parameter condition_prefix Prefix of the class (condition) generated
```
Comment (amousset) [inline tree/30_generic_methods/condition_from_variable_existence.cf]: condition only, not class

File: tree/30_generic_methods/condition_from_variable_existence.cf [hunk 2]
```diff
@@ -18,7 +18,9 @@
 
 # @name Condition from variable existence
 # @description Create a condition from the existence of a variable
-# @documentation This bundle will define a class `{condition_prefix}_{variable_name}_{true/false}
+# @documentation This bundle will define a class `{condition_prefix}_{variable_name}_{true/false}`
```
Comment (amousset) [inline tree/30_generic_methods/condition_from_variable_existence.cf]: condition, not class

**Comment by Fdall** [comment]
Commit modified

**Pull request merged**